soh/src/overlays/actors/ovl_En_Xc/z_en_xc.h

```diff
@@ -10,6 +10,7 @@ typedef void (*EnXcActionFunc)(struct EnXc*, PlayState*);
 typedef void (*EnXcDrawFunc)(struct Actor*, PlayState*);
 
 typedef enum {
+    /*-1 */ SHEIK_TYPE_RANDO = -1,
```

Comment:
I don't think this really makes any real difference, but typically when I've added to source enums I'll add it to the end. Not sure if one is preferable over the other

Reply:
I only really placed it there because of the number ordering.

Reply:
what's the reasoning behind using `-1` instead of `10` here? just curious

Reply:
Weird that I have to do it from a specific tab in GH (thanks Microsoft), but see the comment below.

Reply:
It's possible for the type to be something not in the enum with vanilla play but not negative?

Reply:
I don't think so, but my reasoning was more along the lines of "this is intentionally meant for randomizer." Either -1 or 10 wouldn't matter anyway, as Sheik will kill her actor if the rando enum is used outside randomizer anyway.